python/ometiff_tiler.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+import pyvips
+import argparse
+import os
+
+
+def tile_ometiff(filename, channel_pages, output_directory):
+    for (channel, page) in channel_pages:
+        image = pyvips.Image.tiffload(filename, page=page)
+
+        path = os.path.join(
+            output_directory,
+            'linnarsson.images.{}'.format(channel)
+        )
+
+        if not os.path.exists(path):
+            os.mkdir(path)
+
+        pyvips.Image.dzsave(image, os.path.join(path, channel))
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description='Create tiles with DeepZoom file.')
+    parser.add_argument(
+        '--ometiff_file',
+        help='OME-TIFF image to be tiled.')
+    parser.add_argument(
+        '--channel_page_pairs', required=True, nargs='+',
+        help='Colon-delimited pairs of channels and page values')
```

Comment:
We can check the format of this param: https://docs.python.org/3.7/library/argparse.html#type